sdk/core-shared-tests/src/main/java/com/klaviyo/core_shared_tests/StaticClock.kt

```diff
@@ -1,4 +1,4 @@
-package com.klaviyo.core_shared_tests
+package com.klaviyo.core_shared_tests // ktlint-disable package-name
```

Comment:
ktlint is complaining about underscores in my package names. not sure how this got by pre-commit before, but i need to disable it for now and circle back to it.